README.md

```diff
@@ -130,7 +130,7 @@ An overview of the project folders can be seen below.
 Code references are provided for all platform building blocks. That includes the base library, the VM, a command line application and the compiler. 
 
 * [neo:](https://github.com/neo-project/neo/) Neo core library, contains base classes, including ledger, p2p and IO modules.
-* [neo-modules:](https://github.com/neo-project/neo-modules/) Neo modules include additional tools and plugins to be used with Neo.
```

Comment:
While we're here, we may extend description of `neo:`, because right now it includes more than base classes.

Reply:
Agree, good catch